lib/packagekit-glib2/pk-enum.h

```diff
@@ -523,6 +523,7 @@ typedef enum {
  * @PK_GROUP_ENUM_ACCESSORIES: Accessory packages
  * @PK_GROUP_ENUM_ADMIN_TOOLS: Administration tools packages
  * @PK_GROUP_ENUM_COMMUNICATION: Communication packages
+ * @PK_GROUP_ENUM_DESKTOP_DDE: DDE packages
```

Comment:
```suggestion
 * @PK_GROUP_ENUM_DESKTOP_DDE: Deepin packages
```

Reply:
Same as above, `DDE` is the one we suggested to use.